frontend/src/components/Util/TaskEditors/FloatingTaskEditor.tsx

```diff
@@ -122,12 +122,12 @@ export default function FloatingTaskEditor({
             type={metadata.type}
             icalUID={icalUID}
             taskAppearedDate={taskAppearedDate}
-            mainTask={{ ...mainTask, date: metadata.date }}
-            subTasks={children}
+            taskData={{ ...taskData, date: metadata.date, children }}
```

Comment:
in code above, children is destructured out but not used anywhere, so it's better not separate it from main taskData

Reply:
the main `taskData` here doesn't have a `children` property, so I think we're adding it here so it can match the `TaskMainData` props expected
![image](https://user-images.githubusercontent.com/7517829/98483503-ef832780-21d6-11eb-9fac-e1ba1ca62abe.png)